coast/TIDEGAUGE.py

```diff
@@ -344,20 +345,20 @@ def read_HLW_to_xarray(cls, fn_hlw, date_start=None, date_end=None):
         Returns
         -------
         xarray.Dataset object.
-        """
-        debug(f'Reading "{fn_hlw}" as a HLW file with {get_slug(cls)}')  # TODO Maybe include start/end dates
+        '''
+        debug(f"Reading \"{fn_hlw}\" as a HLW file with {get_slug(cls)}")  # TODO Maybe include start/end dates
         try:
             header_dict = cls.read_HLW_header(fn_hlw)
             dataset = cls.read_HLW_data(fn_hlw, header_dict, date_start, date_end)
-            if header_dict["field"] == "TZ:UT(GMT)/BST":
-                debug("Read in as BST, stored as UTC")
-            elif header_dict["field"] == "TZ:GMTonly":
-                debug("Read and store as GMT/UTC")
+            if header_dict['field'] == 'TZ:UT(GMT)/BST':
+                debug('Read in as BST, stored as UTC')
+            elif header_dict['field'] == 'TZ:GMTonly':
+                debug('Read and store as GMT/UTC')
             else:
                 debug("Not expecting that timezone")
 
         except:
```

Comment:
We should avoid using bare excepts. These will catch **everything**. Including system exits.